en/editor/editor-customization/engine-customization.md

```diff
@@ -0,0 +1,101 @@
+# Engine Customization Workflow
+
+The engine part of Cocos Creator 3.0 includes JavaScript, Cocos2d-x-lite, and adapter (adapter engine customization is not supported at this time). The engine is all open-source on GitHub. The addresses are as follows:
+ 
+- JavaScript engine: <https://github.com/cocos-creator/engine>
+- Cocos2d-x-lite engine: <https://github.com/cocos-creator/cocos2d-x-lite>
+- Adapter: <https://github.com/cocos-creator-packages/adapters>
+
+It is recommended that you maintain your custom code through GitHub's Fork workflow so that you can easily update the custom parts when the engine is upgraded in the future, as described in [Fork a repo](https://help.github.com/articles/fork-a-repo). If you'd like to help Cocos get better and better, feel free to submit your changes to GitHub, see [How to Submit Code to Cocos](../../submit-pr/submit-pr.md). For more GitHub-related workflows, please refer to [GitHub Help](https://help.github.com).
```

Comment:
It is recommended that you maintain your custom code through GitHub's Fork workflow so that you can easily update the custom parts when the engine is upgraded in the future, as described in [Fork a repo](https://help.github.com/articles/fork-a-repo). If you'd like to help Cocos get better and better, feel free to submit your changes to GitHub, see [How to Submit Code to Cocos](../../submit-pr/submit-pr.md). For more GitHub-related workflows, please refer to [GitHub Help](https://help.github.com).
-->
It is recommended to maintain custom code using the GitHub's Fork workflow. This workflow allows developers to easily update custom engine parts when the engine is upgraded in the future. This workflow is described in the [Fork a repo](https://help.github.com/articles/fork-a-repo) documentation. If you would like to help Cocos get better, feel free to submit changes to GitHub, see the [How to Submit Code to Cocos](../../submit-pr/submit-pr.md) documentation. For more GitHub-related workflows, please refer to the [GitHub Help](https://help.github.com) documentation.